03-iam/README.md

```diff
@@ -366,6 +378,8 @@ _Is it possible to limit uploads of objects with a specific prefix (e.g.
 starting with "lebowski/") to an S3 bucket using IAM conditions? If not, how else
 could this be accomplished?_
 
+> It looks like this can be done by inserting a condition in a role policy or appending the prefix to the ARN of the bucket with the action of allow
+
```

Comment:
This cannot be accomplished with a condition because the s3:prefix condition key is [currently only supported by the ListBucket* Actions.](https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazons3.html#amazons3-actions-as-permissions)